README.md

```diff
@@ -121,6 +123,55 @@ In addition, the following ports are provided in this repository:
  - [webassembly](ports/webassembly) -- Emscripten port targeting browsers and NodeJS.
  - [zephyr](ports/zephyr) -- Zephyr RTOS.
 
+Port Tier levels
+----------------
+
+MicroPython operates with a set of Tier levels for the ports. Tiers 1 through 3
+are for ports under active development.  Tier M is for older ports now in
+maintenance mode.  Tier T is for ports used primarily for testing. These Tier
+levels are defined in the table below.
+
+|                                     | Tier 1 | Tier 2 | Tier 3 | Tier M | Tier T |
+|-------------------------------------|--------|--------|--------|--------|--------|
+| actively maintained                 | x      | x      |        | x      | x      |
```

Comment:
It might be easier to make a comparison if we sort this from most stars to least.